### ISSUE TITLE
Support for continuous and discrete color scales in dlx scatter/choropleth

### ISSUE THREAD

**Issue by emilhe** [thedirtyfew/dash-leaflet#46]
At the moment, the built-in examples of pointToLayer/clusterToLayer/style functions support only discrete color scale for choropleth and continuous for scatter and used in the examples. In both cases, continuous and discrete should be supported.

**Comment by emilhe** [comment]
The first attempt at a more general syntax has been implemented in version 0.1.4.rc1,

https://pypi.org/project/dash-leaflet/0.1.4rc1/

In the new syntax, a discrete scale is used if the `classes` argument is provided,

```
import dash
import dash_html_components as html
import dash_leaflet as dl
import dash_leaflet.express as dlx

from numpy import random as rnd

# Setup color stuff.
csc = ['red', 'yellow', 'green', 'blue', 'purple']
clss = [0, 0.2, 0.4, 0.6, 0.8]
ctgs = ["{}+".format(cls, clss[i + 1]) for i, cls in enumerate(clss[:-1])] + ["{}+".format(clss[-1])]
colorbar = dlx.categorical_colorbar(categories=ctgs, colorscale=csc, width=300, height=30)
# Create geojson.
hideout = dict(colorscale=csc, classes=clss)
points = [dict(lat=56 + rnd.uniform(0, 1), lon=10 + rnd.uniform(0, 1), value=rnd.uniform(0, 1)) for i in range(100)]
geojson = dl.GeoJSON(data=dlx.dicts_to_geojson(points), zoomToBounds=True, hideout=hideout,
                     options=dict(pointToLayer=dlx.scatter.point_to_layer))
# Create app.
app = dash.Dash()
app.layout = html.Div([dl.Map([dl.TileLayer(), geojson, colorbar], style={'height': '50vh'})])

if __name__ == '__main__':
    app.run_server(port=7779)
```

Otherwise, it will be continuous.

**Comment by emilhe** [comment]
This update has now been released as part of (0.1.4).
